### PR TITLE
Restore `swap-branches-on-compare`

### LINKED ISSUE #6361: `swap-branches-on-compare` broken
### Description

No link is added here

```
undefined is not an object (evaluating 'icon.parentElement.attributes["aria-label"].value')
```

Screenshot

### How to replicate the issue + URL

https://github.com/refined-github/refined-github/compare/main...23.2.1

### Extension version

23.2.1

### Browser(s) used

safari

### PR DESCRIPTION
**Pull request by kidonng**
Fix #6361

## Test URLs

https://github.com/refined-github/refined-github/compare/23.2.1...main

## Screenshot

截屏2023-04-06 19 52 55

### CODE REVIEW

**Comment by fregante** [comment]
Thanks for the PR!

I noticed you sent this before my last comment. How do you feel about this?

Screenshot 10

Or maybe `.btn-octicon` to match the other icon colors:

Screenshot 12

File: source/features/swap-branches-on-compare.tsx [hunk 1]
```diff
@@ -18,19 +17,34 @@ function init(): void {
...
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isCompare,
 	],
 	exclude: [
+		// Disable on Two-dot Git diff comparison #4453
 		() => /\.\.+/.exec(location.pathname)?.[0]!.length === 2,
+		// Prefer the native "switch base" button if it is available
```
Comment (fregante) [inline source/features/swap-branches-on-compare.tsx:36]: Is there a native one? When does it appear? Maybe we should just reuse its style

Comment (kidonng) [reply source/features/swap-branches-on-compare.tsx:36]: Oops, not button but link (it's the one in my first screenshot)

Comment (fregante) [reply source/features/swap-branches-on-compare.tsx:36]: Oh right

**Comment by kidonng** [comment]
Yeah a button works too, in that case there's no need to hide it.

**Review by fregante — APPROVED** [review]
LGTM, I'm just afraid that it steals too much attention with a regular button style. Don't like `btn-octicon`?

**Comment by kidonng** [comment]
Because it is... not an icon? Looks too weird and not something clickable to me.

**Comment by grahamperrin** [comment]
Hey, thanks! 

A few times, before this feature reappeared, I clicked the three dots beneath the arrow (between the two things to be compared), imagining that a menu would appear with _Swap_ amongst its options. Oops.

**Comment by kidonng** [comment]
The three dots link is a nice touch nonetheless, it just _happened_ to break our feature.

**Pull request merged**